src/ecs/commands/runCommandInContainer.ts

```diff
@@ -127,16 +48,34 @@ export async function runCommandInContainer(
             throw Error('SSM Plugin not installed and cannot auto install')
         }
 
-        const execCommand = await node.ecs.executeCommand(node.clusterArn, node.containerName, task, command)
+        const execCommand = await node.ecs.executeCommand(
+            node.clusterArn,
+            node.containerName,
+            response.task,
+            response.command
+        )
         const args = [JSON.stringify(execCommand.session), node.ecs.regionCode, 'StartSession']
+        showOutputMessage(
+            `${moment().format(LOCALIZED_DATE_FORMAT)}  Container: "${node.containerName}" Task ID: "${
+                response.task
+            }"  Running command: "${response.command}"`,
+            outputChannel
+        )
+
+        window.showInformationMessage(
+            localize(
+                'AWS.command.ecs.runCommandInContainer.runningCommandMessage',
+                'Running command. This may take several minutes.'
+            )
+        )
         const cp = await new ChildProcess(ssmPlugin, args).run()
         if (cp.exitCode !== 0) {
             result = 'Failed'
-            showOutputMessage(cp.stderr, outputChannel)
+            showOutputMessage(removeAnsi(cp.stderr), outputChannel)
             throw cp.error
         } else {
             result = 'Succeeded'
-            showOutputMessage(cp.stdout, outputChannel)
+            showOutputMessage(removeAnsi(cp.stdout), outputChannel)
         }
```

Comment:
Nit: is it worth showing both stderr and stdout no matter the situation? Error messages aren't just in stderr, and output isn't always limited to stdout.